e2e/e2e-example.sh

```diff
@@ -1,19 +1,19 @@
 #!/bin/bash
 # Copyright 2022 The OWASP Coraza contributors
 # SPDX-License-Identifier: Apache-2.0
-# Copied from https://github.com/corazawaf/coraza-proxy-wasm/tree/main/e2e
-ENVOY_HOST=${ENVOY_HOST:-"localhost:8080"}
+# Revisited from https://github.com/corazawaf/coraza-proxy-wasm/tree/main/e2e
```

Comment:
In a next effort we should port this tests to go an reuse across different connectors.